### PR TITLE
Fix typings and dependencies

### PR DESCRIPTION
**Pull request by Zyie**
Kind of went down a bit of a rabbit hole when looking at changing the dev dependencies for pixi based on the comment in the other PR

This PR updates the pixi dev dependency but also fixes some type issues!

We can close #3 if you are happy with these changes

### CODE REVIEW

File: packages/storybook-renderer/src/index.ts
Comment (baseten) [inline packages/storybook-renderer/src/index.ts:1]: I remember adding this to fix something, I'm assuming you have no errors?

**Review by baseten — APPROVED** [review]
LGTM 👍 

I only ever copied/tweaked the types from the original HTML renderer so don't entirely have a handle on these changes, but assuming everything is working nicely for you, go ahead!

**Comment by baseten** [comment]
Hey @Zyie, did you want me to do the merge? No problem if so, just used to the flow where person who opens the PR merges the PR after approval 😄

**Comment by Zyie** [comment]
> Hey @Zyie, did you want me to do the merge? No problem if so, just used to the flow where person who opens the PR merges the PR after approval 😄

I'm very much used to the opposite 😄 

I can merge but could you do a release when you have the time!

**Comment by baseten** [comment]
> I'm very much used to the opposite 😄

Haha, maybe this is more the open source way? Maintainers merge in PRs vs projects where devs shepherd their PRs from review to production? 😄

**Pull request merged**
